buildenv/jenkins/JenkinsfileBase

```diff
@@ -899,6 +899,7 @@ def post(output_name) {
 				def pattern = "${env.WORKSPACE}/*_output.*"
 				uploadToArtifactory(pattern)
 			}
+			archiveFile("jvmtest/perf/**/leftoverResults/*/*", false)
```

Comment:
Do we need `*/*`?

Reply:
the first * is timestamp folder, and second * is for all files under it. two layers.

Reply:
We should achieve leftoverResults dir regardless of the subdirs. Can you test if we need to specify `*/*`?